src/components/AccountActivitiesTable.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <table class="account-activities-table">
+    <thead>
+      <tr>
+        <th>
+          Hash
+          <hint-tooltip>
+            {{ accountHints.activityHash }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Time
+          <hint-tooltip>
+            {{ accountHints.activityTime }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Type
+          <hint-tooltip>
+            {{ accountHints.activityType }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Activity
+          <hint-tooltip>
+            {{ accountHints.activityMeaning }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Data
+          <hint-tooltip>
+            {{ accountHints.activityData }}
+          </hint-tooltip>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr
+        v-for="(activity, index) in accountActivities?.data"
+        :key="`${activity.hash}-${activity.type}-${index}`">
```

Comment:
```suggestion
        :key="index">
```
Index on its own should do the job

Reply:
I did it this way because it's not a good practice to use the index alone https://stackoverflow.com/questions/44531510/why-not-always-use-the-index-as-the-key-in-a-vue-js-for-loop

Reply:
Aha, thanks. I was not aware of that. So the nature of data is live at and any moment new record can pop up in the table and break the keys, right?

Reply:
Well... it's not live in the sense that it's fetched through WebSockets but loading other pages will put another set of records into the array with the same indexes but different content

Reply:
Would this be solved by resetting store data before fetching new ones as I pointed out below?

Reply:
I used that long key out of the habit of not setting an index as the key. In this specific example, index for the key is just fine for the reason you mentioned so I simplified it.

https://vuejs.org/api/built-in-special-attributes.html#key